### ISSUE TITLE
update to bootstrap 4.0.0-beta.3

### ISSUE THREAD

**Issue by patkub** [patkub/rmc1891-site#71]
https://getbootstrap.com/docs/4.0/migration/#beta-3-changes

**Comment by patkub** [comment]
Getting an issue compiling beta 3 sass
```
Message:
    node_modules\bootstrap\scss\_input-group.scss
Error: ".input-group-lg > .form-control" failed to @extend ".form-control-lg".
       The selector ".form-control-lg" was not found.
       Use "@extend .form-control-lg !optional" if the extend should be able to fail.
        on line 121 of node_modules/bootstrap/scss/_input-group.scss
>>   @extend .form-control-lg;
   ----------^
```

**Comment by patkub** [comment]
This is fixed in bootstrap 4.0.0 release
